lighthouse-core/scripts/generate-bundlephobia-database.js

```diff
@@ -64,12 +64,12 @@ function hasBeenRecentlyScraped(library) {
  * @return {library is BundlePhobiaLibrary}
  */
 function validateLibraryObject(library) {
-  return library.hasOwnProperty('name') &&
-    library.hasOwnProperty('size') &&
-    library.hasOwnProperty('gzip') &&
-    library.hasOwnProperty('description') &&
-    library.hasOwnProperty('repository') &&
-    library.hasOwnProperty('version') &&
+  return typeof library.name === 'string' &&
+    typeof library.size === 'number' &&
+    typeof library.gzip === 'number' &&
+    typeof library.description === 'string' &&
+    typeof library.repository === 'string' &&
+    typeof library.version === 'string' &&
```

Comment:
this lint rule is for [preventing risks due to prototype pollution](https://eslint.org/docs/rules/no-prototype-builtins) that's has no bearing on us here, but I'm also not sure what the intention of using `hasOwnProperty` was. Checking `typeof` seems more straightforward and is stricter to boot.